Current PRs / Actions failing for msys2 and SVGTiny

Seems like svgtiny is failing now even though there haven't been any changes to the formula. 
What is weird is it has been working and passes in the master branch but any current CI runs errors ( for all msys2 / ucrt64 etc platforms ). @oxillo any chance you could take a look?

from: https://github.com/openframeworks/apothecary/actions/runs/3433644522/jobs/5724183530 
```
cc -c src/svgtiny_gradient.c -o src/svgtiny_gradient.o -O2 
src/colors.gperf:17:10: fatal error: svgtiny.h: No such file or directory
   17 | #include "svgtiny.h"
      |          ^~~~~~~~~~~
compilation terminated.
cc -c src/svgtiny_list.c -o src/svgtiny_list.o -O2 
src/svgtiny.c:17:10: fatal error: dom/dom.h: No such file or directory
   17 | #include <dom/dom.h>
      |          ^~~~~~~~~~~
compilation terminated.
src/svgtiny_gradient.c:14:10: fatal error: svgtiny.h: No such file or directory
   14 | #include "svgtiny.h"
      |          ^~~~~~~~~~~
compilation terminated.
make: *** [Makefile:72: src/autogenerated_colors.o] Error 1
make: *** Waiting for unfinished jobs....
make: *** [Makefile:72: src/svgtiny.o] Error 1
make: *** [Makefile:72: src/svgtiny_gradient.o] Error 1
src/svgtiny_list.c:15:10: fatal error: svgtiny.h: No such file or directory
   15 | #include "svgtiny.h"
      |          ^~~~~~~~~~~
compilation terminated.
```

Thanks! 
I just tried it here and still get an error though it it is different now. 

https://github.com/openframeworks/apothecary/actions/runs/3452128137/jobs/5761823592 

```
cc -c libwapcaplet/src/libwapcaplet.c -o libwapcaplet/src/libwapcaplet.o -O2 "-I/d/a/apothecary/apothecary/scripts/../out/libxml2/include"  -D_BSD_SOURCE -D_DEFAULT_SOURCE -DWITHOUT_ICONV_FILTER -fPIC -I/d/a/apothecary/apothecary/apothecary/build/svgtiny/include/ -I/d/a/apothecary/apothecary/apothecary/build/svgtiny/src -Wall -W -Wundef -Wpointer-arith -Wwrite-strings -Wstrict-prototypes -Wmissing-prototypes -Wmissing-declarations -Wnested-externs -std=c99 -Ilibdom/include -Ilibdom/src -Ilibdom/bindings -Ilibwapcaplet/include -Ilibparserutils/include -Ilibparserutils/src -Ilibxml2/include
[683](https://github.com/openframeworks/apothecary/actions/runs/3452128137/jobs/5761823592#step:4:684)
libdom/bindings/xml/libxml_xmlparser.c:12:10: fatal error: libxml/parser.h: No such file or directory
```

--- a/.github/workflows/build-msys2.yml
+++ b/.github/workflows/build-msys2.yml
@@ -35,13 +35,13 @@ jobs:
             base-devel
             unzip
             dos2unix
-            gperf
             git
-            libxml2
-            python3
           pacboy: >-
             gcc:p
             cmake:p
+            gperf:p
+            libxml2:p
+            python3:p
       - name: Clone repository
         uses: actions/checkout@v2
       - name: Build

